src/nspawn/meson.build

```diff
@@ -35,7 +35,23 @@ libnspawn_core = static_library(
                         userspace],
         build_by_default : false)
 
-systemd_nspawn_sources = files('nspawn.c')
+nspawn_libs = [
```

Comment:
Why not inline this?

Reply:
As it will be used later by the commits waiting in #28042.